docs/standard/serialization/system-text-json/how-to.md

```diff
@@ -43,6 +50,8 @@ The preceding examples use type inference for the type being serialized. An over
 :::code language="csharp" source="snippets/how-to/csharp/SerializeWithGenericParameter.cs" highlight="23":::
 :::code language="vb" source="snippets/how-to/vb/RoundtripToString.vb" id="SerializeWithGenericParameter":::
 
+You can also use GitHub Copilot to generate serialization code for you. For instructions, see the [Use GitHub Copliot](#use-github-copilot-to-serialize-to-json) section in this article.
```

Comment:
I didn't fully notice this before. Do we really need a tip on both line 24 and line 53? Also there's a spelling mistake in Copliot.

Reply:
Fixed typo, thanks for catching that. With the code snippets in between in the article, the mention again doesn't seem too close to each other. Also, here it's mentioned in the context of where to use it. The one in the intro is general tip.